classes/teacher.rb

```diff
@@ -0,0 +1,14 @@
+require './person'
```

Comment:
- [ ] **[OPTIONAL]** Based on ruby conventions, I would recommend you use `require_relative` instead.

Reply:
Done.